actions/class.Runner.php

```diff
@@ -306,6 +314,27 @@ public function init()
         $this->returnJson($response, $code);
     }
 
+    private function guessThemeNamespace($executionUri)
+    {
+        //This part is rather questionable as it creates a dependency from LTI
+//        $currentSession = \common_session_SessionManager::getSession();
+//        if ($currentSession instanceof TaoLtiSession) {
+//            $launchData = $currentSession->getLaunchData();
+//            if ($launchData->hasVariable(LtiThemeSwitcher::LTI_VARIABLE)) {
+//                $config["activeNamespace"] = $launchData->getVariable(LtiThemeSwitcher::LTI_VARIABLE);
+//            }
+//        }
+
+        /** @var \oat\taoDelivery\model\execution\DeliveryExecution $deliveryExecution */
+        $deliveryExecution = ServiceProxy::singleton()->getDeliveryExecution($executionUri);
+
+        $delivery = $deliveryExecution->getDelivery();
+
+        $themeNamespace = $delivery->getOnePropertyValue(new Property(DeliveryThemeDetailsProvider::DELIVERY_THEME_ID_URI));
```

Comment:
@jbout don't we try to limit the access to the ontology during a test ?

Reply:
Correct, already discussed this with him, this should not be done here